CHANGES.rst

```diff
@@ -6,6 +6,9 @@
   - Add ``:working-directory`` option to ``flycheck-define-command-checker``
     [GH-973] [GH-1012]
 
+  - ``flycheck-go-build-install-deps`` turns on dependency installation for ``go test``
```

Comment:
Sorry, I forgot one thing: Could you please remove the blank line above this line?  In ReST there shouldn't be blank lines within lists, only between different nesting levels.